libs/galley-types/src/Galley/Types/Swagger.hs

```diff
@@ -240,6 +240,12 @@ conversationAccessUpdate = defineModel "ConversationAccessUpdate" $ do
     property "access" (unique $ array bytes') $
         description "List of conversation access modes: []|[invite]|[invite,code]"
 
+conversationReceiptModeUpdate :: Model
+conversationReceiptModeUpdate = defineModel "conversationReceiptModeUpdate" $ do
+    description "Contains conversation receipt mode to update"
```

Comment:
The description doesn't give more information than the name; please explain the concept here for client developers (e.g. reuse part of the comment for the `newtype ReceiptMode`)